src/postgraphql/withPostGraphQLContext.ts

```diff
@@ -135,6 +140,14 @@ async function setupPgClientTransaction ({
   // Sql query.
   const localSettings = new Map<string, mixed>()
 
+  // Set the custom provided settings before jwt claims and role are set
+  // this prevents an accidentional overwriting
+  if (typeof pgSettings === 'object') {
+    for (const key of Object.keys(pgSettings)) {
+      localSettings.set(key, pgSettings[key])
```

Comment:
Please wrap the value with String constructor: localSettings.set(key, String(pgSettings[key]))